### ISSUE TITLE
The implementation of the slide master is missing

### ISSUE THREAD

**Issue by ghost** [ONLYOFFICE/DocumentServer#2144]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Slide master implementation is required for master editing.  need to implement outline or pattern mode in documents, presentations, and spreadsheets

**Comment by MichaelFA7** [comment]
Yes, since it is very necessary to implement a pattern in documents, presentations, spreadsheets, etc. since it is for editing in design mode, such as editing the background of the file or document

**Comment by MichaelFA7** [comment]
and I would like that when creating QR and bar codes they were editable, such as color, various styles of size, design and others

**Comment by Tarek-Hasan** [comment]
LibreOffice supports creating or reading QR code and Barcode for many years now. They use [C++ port](https://github.com/zxing-cpp/zxing-cpp) of [ZXing library](https://github.com/zxing/zxing). ZXing (Zebra Crossing) barcode scanning library has been used by many android apps for a decade.

**Comment by Tarek-Hasan** [comment]
Any update on progress of QR code and Barcode implementation?

**Comment by Tarek-Hasan** [comment]
Any update on progress of QR code and Barcode implementation?

**Comment by maruacat** [comment]
> Any update on progress of QR code and Barcode implementation?

Hello @Tarek-Hasan!
Sorry, but now I don't have any news about the implementation of QR code and barcode in our editor."
Have a good day!

**Comment by Tarek-Hasan** [comment]
There is now a qr code plugin in OnlyOffice.

**Comment by trofim24** [comment]
Slide master editing has been implemented in the Presentation Editor - it was added in version **8.1.0** ("Added a mode for editing master slides and templates") and has been improved in later releases. 

Full details are in the [CHANGELOG](https://github.com/ONLYOFFICE/DocumentServer/blob/master/CHANGELOG.md). Please update to the latest version to try it.

@Tarek-Hasan We've also reopened #3779 to track QR codes and barcodes as a separate feature request - please follow that issue for updates on it.

Since the requested functionality is available, we're closing this issue. If something specific is still missing, feel free to open a new issue with the details.
